crates/rolldown_utils/src/sanitize_identifier.rs

```diff
@@ -0,0 +1,31 @@
+use std::borrow::Cow;
+
+use oxc::syntax::keyword::RESERVED_KEYWORDS;
+
+// Follow from https://github.com/rollup/rollup/blob/master/src/utils/identifierHelpers.ts#L17-L25
+#[allow(clippy::needless_pass_by_value)]
+pub fn sanitize_identifier(str: Cow<str>) -> String {
```

Comment:
There already exists a util function here https://github.com/rolldown/rolldown/blob/2ec340edda3da25c3be15562335b62d1a8a787f8/crates/rolldown_utils/src/ecmascript.rs?plain=1#L14-L49

Reply:
Thanks! I'm looking for it